tests/unit/test_charm.py

```diff
@@ -93,7 +93,7 @@ def test_on_leader_elected_secrets(self):
         secret_data = self.harness.model.get_secret(label="mysql-k8s.app").get_content()
 
         # Test passwords in content and length
-        required_passwords = ["root-password", "server-config-password", "cluster-admin-password"]
```

Comment:
Nothing about his in PR description. Q: Why?

Reply:
forgot to pop from local stash, fixed ea1e96d